### DISCUSSION TITLE
rspamd included in Virusbulletin tests

### DISCUSSION THREAD

**Discussion by ghost** [rspamd/rspamd#4357] [General]
Hi!

I have seen that rspamd is regularly testet in the virusbulletin test:
https://www.virusbulletin.com/virusbulletin/2022/09/vbspam-comparative-review/

Does somebody know with what kind of configuration they are testing?

Best Regards
Dominik

**Comment by moisseev** [comment]
I have a few more questions:

1. Which product versions the compare?
2. How did they train Rspamd before testing.
3. Why do they think there is no DKIM, SPF, WebGUI, API support in Rspamd?
